nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml

```diff
@@ -710,8 +784,6 @@
                         <configuration>
                             <packagingIncludes>
                                 assets/**/*,
-                                css/reset.css,
-                                css/reset.css.gz,
                                 css/common-ui.css,
                                 css/common-ui.css.gz,
                                 css/message-page.css,
```

Comment:
We need to add the slick-nifi-theme here:

css/slick-nifi-theme.css,
css/slick-nifi-theme.css.gz,